api/controllers/CommunityController.js

```diff
@@ -118,7 +119,7 @@ module.exports = {
 
   joinWithCode: function (req, res) {
     var community
-    return Community.query('whereRaw', 'lower(beta_access_code) = lower(?)', req.param('code')).fetch()
+    return Community.query('whereRaw', 'lower(beta_access_code) = lower(?) and active=true', req.param('code')).fetch()
```

Comment:
for the sake of cleanliness, can you switch this to the function syntax and have the active=true bit in a normal where?